package.json

```diff
@@ -17,7 +17,8 @@
   ],
   "scripts": {
     "lint:all": "yarn workspaces run lint",
-    "format:all": "yarn workspaces run format"
+    "format:all": "yarn workspaces run format",
```

Comment:
we can probably remove the `:all` prefix now since its implied when run in the workspace root